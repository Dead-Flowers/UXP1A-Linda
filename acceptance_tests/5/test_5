-t 1 output '(\"abc\")'
-t 1 output '(\"a\")'
-t 1 output '(\"ab\")'
-t 1 output '(\"ac\")'
-t 1 output '(\"acc\")'
-t 5 input '(string:>\"a\")'
-t 5 input '(string:>\"a\")'
-t 5 input '(string:>\"a\")'
-t 5 input '(string:>\"a\")'
-t 5 input '(string:>\"a\")'
