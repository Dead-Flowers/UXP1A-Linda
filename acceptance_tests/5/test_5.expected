Exit code: 0
Exit code: 0
Exit code: 0
Exit code: 0
Exit code: 0
("abc")
Exit code: 0
("acc")
Exit code: 0
("ac")
Exit code: 0
("ab")
Exit code: 0
Exit code: 2
